RNTester/android/app/build.gradle

```diff
@@ -146,6 +165,10 @@ dependencies {
     // Build React Native from source
     implementation project(':ReactAndroid')
 
+    def hermesPath = '$projectDir/../../../../node_modules/hermesvm/android/'
+    debugImplementation files(hermesPath + "hermes-debug.aar")
```

Comment:
should this be wrapped in an `if (enableHermes)`? We don't need both hermes and jsc.